openedx/openedx-events
# Dmh/persistent grade event

**Description:** 

This PR adds a new event that is going to trigger when a persisted grade is created or updated.

**Dependencies:**

This PR is a dependency for the following PR <link pr edx-platform>

**Merge deadline:** 

As soon as possible

**Installation instructions:**

1. Make sure you are using the changes on the following edx-platform PR, these changes contain the code with the signal to test it.
2. Install the current openedx-events changes. 
3. Set True the DEBUG setting in lms/envs/devstack.py to visualize the log when the event is triggered.

**Testing instructions:**

1. Create a course with some graded units.
2. Submit an answer in the course.
3. Check out your lms logs, there will be a message indicating that the event has been sent.
example: Responses of the Open edX Event <org.openedx.learning.course.persistent_grade.summary.v1>

**Reviewers:**


**Merge checklist:**


**Post merge:**


**Author concerns:**
None
